Article/Article.js

```diff
@@ -1,46 +1,19 @@
-// Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
-
 class Article {
```

Comment:
looks like you got the expand to work on this Article class. comparing this to the older commits I can't really see where that issue was coming from. do you know why it wasn't working the way you intend it to?